tutorials/getting_started/rules_basic.md

```diff
@@ -0,0 +1,196 @@
+
+![basic rule YAML modified](images/basic-rule-yaml-modified.png)
+
+This will work for any UI Rule, not just basic rules.
+
+Note, when asking for help on the forum, the experts will want to see this code version pasted as text rather than a screen shot of either the Design tab or the Code tab.
+
+## Rule Templates
+
+At some point basic rules will not be sophisticated enough to reach your goal.
+The first place to look when that is the case are the rule templates posted to the Marketplace.
+You can find them under Settings and Automation.
+
+### Installation and Configure
+
+Under the "Languages & Technologies" section there are "Rule Templates" and "Blockly Libraries" sections (discussed on the next page).
+These are community created capabilities posted to the [Marketplace](https://community.openhab.org/c/marketplace/69).
+Someone may have already written the code for you so all you have to do is install and configure it.
+
+In MainUI click on a rule template that seems to do what you want.
+This will open it's documentation page which explains how the template works, configuration parameters, dependencies, etc.
+
+![alarm clock rule template](images/alarm-clock-rule-template.png)
+
+To install the template, click on "Add" next to the template's logo.
+If installation was successful, you can now create one or more rules based on the template.
+
+### Rule Template Instantiation
+
+Navigate to Rules and click the blue + icon to create a new Rule.
+
+Fill out the Rule's metadata as usual.
+However, now you will see a "Create from Template" section listing all the installed templates.
+Choose the template you want to base this rule on.
+
+![create rule from template](images/rule-create-from-template.png)
+
+The bottom of the form will change from the three sections described previously to a form showing the parameters the rule template defines.
+
+![configure rule](images/rule-template-config.png)
+
+In this case, the parameters are selecting a DateTime Item to trigger the rule and which defines the "alarm time" and a "Script" to call when at the date and time in that DateTime Item.
+
+### Updates to Templates
+
+Your rule, once instantiated becomes its own thing separated from the template, just as if you had cut and paste the code as in the first example.
+Therefore, it will not change even if the rule template is changed or removed.
+This means the rule will not update based on changes to the template automatically.
+To update a template and a rule use the following process.
+
+1. Notice that there has been an update to a template that you want to use (keep checking back).
+1. Select the template in MainUI and click "Remove". Then click "Add" to refresh the template with the new changes.
+1. Go to Settings and Rules and navigate to the rule you want to update. Click the pause icon to disable that rule. Or if you don't want to keep a backup, delete it. Note, if you just disable you won't be able to use the same rule UID for the new rule.
```

Comment:
ordered lists can handle one sentence per line writing too, so you could split this up and have a look at the preview.